experiments/ClimaEarth/components/ocean/prescr_ocean.jl

```diff
@@ -0,0 +1,116 @@
+import ClimaUtilities.TimeVaryingInputs: TimeVaryingInput, evaluate!
+import ClimaUtilities.ClimaArtifacts: @clima_artifact
+import Interpolations # triggers InterpolationsExt in ClimaUtilities
+import Thermodynamics as TD
+import ClimaCoupler: Checkpointer, FieldExchanger, Interfacer
+
+"""
+    PrescribedOceanSimulation{P, Y, D, I}
```

Comment:
```
PrescribedOceanSimulation{I}
```
What does `I` stand for?

Reply:
I just copied this over from the SurfaceStub definition. I can change it to `C` if you prefer